packages/@webex/plugin-meetings/src/peer-connection-manager/util.js

```diff
@@ -1,6 +1,32 @@
+import {parse} from '@webex/ts-sdp';
```

Comment:
is possible convert to ts file

Reply:
Sure, @arun3528, we will convert that and commit the code. thanks.
@meta-dipanshu-sharma, @rajeshtezu90

Reply:
@arun3528, Incorporated the comments and committed the changes.